db/app.controller.js

```diff
@@ -59,3 +57,17 @@ selectArticlesById(article_id).then((article)=>{
     next(err)
 })
 }
+exports.addNewCommentByArticleId =(req,res,next) => { 
+const newComment =req.body
+const {article_id} =req.params
+const checkArticleExists = selectArticlesById(article_id) 
+const insetCommentIfExits =insertNewComment(newComment,article_id)
+
+Promise.all([checkArticleExists,insetCommentIfExits])
+.then(result => {
+    res.status(201).send(result[1].rows[0])
```

Comment:
Remember to send back your data on a descriptive key, in this case { comment : `data` }